webapp/src/components/root_portal.tsx

```diff
@@ -12,6 +12,14 @@ type Props = {
     children: React.ReactNode,
 }
 
+export class InjectionProvider extends React.Component<any> {
+    public render(): JSX.Element {
+        const stores = {...this.props};
+        delete stores.children;
+        return React.createElement(Provider as any, stores, this.props.children);
+    }
+}
```

Comment:
Why is this needed?

Reply:
It fixes this one

![image](https://user-images.githubusercontent.com/29619100/165040801-9a9084e0-1ea9-45d6-b585-8d2d3ea16d99.png)

Reply:
It seems like there should be a cleaner solution for a lint issue like this 🤷 Any thoughts on how this change may introduce a bug? Was this solution sourced from somewhere on the web?

Reply:
I get the solution here https://github.com/mobxjs/mobx-react/issues/342#issuecomment-357929369

Reply:
@jespino Curious what you think about this. Probably insignificant